docs/_documentations/troubleshooting.md

```diff
@@ -384,13 +384,13 @@ Sometimes when a new project is created, it doesn't show up in the hierarchy vie
 <!--
 Action/Topic: Creating a project and/or Checking the application and build statuses
 Issue type: bug/info
-Issue link:
-18.10:
+Issue link: Updated in https://github.com/eclipse/codewind/issues/2613
+0.11.0: Updated for this release.
 -->
 ## Context Root / Application Endpoint not correct
 If you create or bind a project that has a context root set in `.cw-settings`, such as a project using the Lagom template, the context root is not picked up initially. This also happens after restarting Codewind.
 
-**Workaround** For Eclipse, add the context root to the URL in your browser. For example, the browser might open with `localhost:34567` instead of `localhost:34567/mycontextroot`, so type `mycontextroot`. For VS Code and Che, edit and save the `.cw-settings` file, and the context root updates.
+**Workaround** For a permanent fix for all IDEs, edit and save the `.cw-settings` file, and the context root updates. For a temporary workaround for Eclipse, add the context root to the URL in your browser. For example, the browser might open with `localhost:34567` instead of `localhost:34567/mycontextroot`, so type `mycontextroot`. 
```

Comment:
The temporary workaround applies to all IDEs too.

Reply:
Adjusted to `For a temporary workaround, ...`